elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -1,4 +1,15 @@
 {{#question}}
+<script>
+$(function() {
+    var UUID = "{{uuid}}";
+
+    new window.PLOrderBlocks(UUID, {
+        maxIndent: {{max_indent}},
+        enableIndentation: {{#check_indentation}}true{{/check_indentation}}{{^check_indentation}}false{{/check_indentation}}
```

Comment:
This isn't that pretty, is there a better way to do this with moustache? I tried just directly including the variable but it dropped it in capitlized Python-style which obviously didn't work for JavaScript.

Reply:
If there isn't a nice mustache construct for it, I could just do something like
```
"true" if check_indentation else "false"
```
in the python, which is still not that cool but is probably prettier than this.

Reply:
`"true" if condition else "false"` is the pattern we use in other elements - I think that's totally fine to use here as well.